internal/sdk/external_tables_dto_builders_gen.go

```diff
@@ -2,8 +2,6 @@
 
 package sdk
 
-import ()
```

Comment:
this is generated we should not touch it

Reply:
okay but our linter doesn't like it